lifetimes/plotting.py

```diff
@@ -662,16 +740,35 @@ def plot_dropout_rate_heterogeneity(
     fig, ax = plt.subplots(1)
     fig.suptitle(suptitle, fontsize=suptitle_fontsize, fontweight="bold")
 
-    ax.set_title("mean: {:.3f}, var: {:.3f}".format(beta_mean, beta_var))
+    ax.set_title("{}: {:.3f}, {}: {:.3f}".format(title_list[0], rate_mean, title_list[1], rate_var))
```

Comment:
here's the problem: `rate_mean` is not defined

Reply:
Ah, yes, stupid copy paste problem. I'll fix it tomorrow.

Reply:
(Now it's fixed, I guess?)